github/github.go

```diff
@@ -366,6 +368,11 @@ func EvaluatePullRequest(logger *zap.Logger, postgres db.IClaDB, evalInfo *types
 			return err
 		}
 
+		err = createRepoStatus(client.Repositories, evalInfo.RepoOwner, evalInfo.RepoName, evalInfo.Sha, "failure", "One or more commits haven't met our Quality requirements.", botName)
```

Comment:
I'm looking into trying to use the Mock objects to see if we can assert some state in unit tests for these sorts of things. It might complicate this PR a little, but should hopefully open the door to better unit tests and maybe allow refactoring some of these huge methods into smaller units. Spoiler: `repositoriesMock` yadda, yadda...